server/src/main/java/org/opensearch/indices/ShardLimitValidator.java

```diff
@@ -66,6 +68,14 @@ public class ShardLimitValidator {
         Setting.Property.NodeScope
     );
 
+    public static final Setting<Integer> SETTING_CLUSTER_MAX_SHARDS_PER_CLUSTER = Setting.intSetting(
```

Comment:
Since we now have 2 settings to limit the shards count, we have to ensure that they are not contradictory, please add validation rules that at least SETTING_MAX_SHARDS_PER_CLUSTER >= SETTING_CLUSTER_MAX_SHARDS_PER_NODE

Reply:
Addressed the comment.